README.md

```diff
@@ -120,18 +142,26 @@ browse order history, etc.
 Once you are finished, you can clean up the environment by executing
 
 ```bash
-$ kubectl delete -k k8s/core 
+$ kubectl delete -k k8s/core -n sockshop-${SOCKSHOP_BACKEND} 
```

Comment:
Should this be
```
kubectl delete -k k8s/${SOCKSHOP_BACKEND} -n sockshop-${SOCKSHOP_BACKEND}
```
instead?

I would also move it from here to the end of the "Cleanup" section below.